### PR TITLE
Add a command line interface for track conversion

### PR DESCRIPTION
**Pull request by K-Meech**
This PR adds a `click` command line interface for the track conversion code (+ tests + documentation on how to use it)

See the nicely formatted documentation [on the track_converter_cli branch](https://github.com/K-Meech/Lineage-analysis/blob/km/track_converter_cli/docs/tracking_formats.md) (CLI information is at the bottom)

@pembacher - let me know if anything needs changing here, or if you want to discuss any of the code changes

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,8 @@ classifiers = [
 ]
 dependencies = [
   "pandas",
-  "btrack"
+  "btrack",
+  "click"
```
Comment (giordano) [inline pyproject.toml]: Comment that I also wanted to make in #12: I think it'd be better to specify compatibilities, like I did in #2 at https://github.com/Jamie-Dean-Lab/Lineage-analysis/blob/7f21b6285a965fd8234714fc8a3f44f94c36b2d7/pyproject.toml#L21

Side note, always leaving a trailing comma in these lists allows you not to modify the previous line when you want to add a new item, which keeps git history (and git blame in particular) cleaner :wink:

**Review by giordano — APPROVED** [review]
I didn't test this myself, but I had a look at the code and looks good to me, thanks! I only have a minor comment about dependencies in `pyproject.toml`

File: pyproject.toml (hunk 1 above)
Comment (K-Meech) [reply pyproject.toml]: Thanks @giordano ! I've pinned all the main dependencies to `>=latest_version <next_major_version` (except for the dev dependencies, as these are optional and only needed for testing/linting). I also added a trailing comma :smile:

**Comment by K-Meech** [comment]
@pembacher - feel free to merge this PR if you're happy with it :smile:

**Pull request merged**